### ISSUE TITLE
The examples should work - should they not? They should at least compile.

### ISSUE THREAD

**Issue by tabinnorway** [MicrosoftDocs/msteams-docs#1484]
Neither does. Download the C# example, open in VS - nothing works. Lots of errors. Try to upgrade NuGet packages and things get worse.
A little better with the Node example: Hoek.inherits is not a function. OK, we can probably track this down easily enough.

Teams is a popular application. The QA for the examples is not up to par.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c4fa27cb-544f-f8a2-97a7-4b31528c1b80
* Version Independent ID: fa69cd6f-1ef4-706d-2816-aa709714b0aa
* Content: [Get started with App Studio and Node.js - Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/tutorials/get-started-nodejs-app-studio)
* Content Source: [msteams-platform/tutorials/get-started-nodejs-app-studio.md](https://github.com/MicrosoftDocs/msteams-docs/blob/master/msteams-platform/tutorials/get-started-nodejs-app-studio.md)
* Product: **msteams**
* GitHub Login: @laujan
* Microsoft Alias: **lajanuar**

**Comment by Trinetra-MSFT** [comment]
@tabinnorway , Sorry for your inconvenience, Could you please take a look at latest [Bot Builder SDK V4](https://github.com/microsoft/BotBuilder-Samples/tree/master/samples) sample code? Let me know if this doesn't help you out.

**Comment by kosset** [comment]
@tabinnorway Regarding the NodeJS sample, try uninstalling gulp from global scope `npm uninstall -g gulp`, then re-install all the dependencies from the sample-project `npm install` and run the app `npm start`.

I had the same problem and this worked for me. Cheers :)

**Comment by aaronrogers** [comment]
@kosset, I've tried that and I get a whole slew of errors.  If I delete `node_modules` and `package-lock.json` I get farther as everything installs with `npm install`.  However, when I run `npm start` I get `sh: nodemon: command not found`.  So I install nodemon. `npm install --save-dev nodemon`  Then with `npm start` I get

> Warning: To load an ES modeul, set "type": "module" in the package.json or use the .mjs extension.
> app.js:4
> import path from 'path';

**Comment by mfbhatt** [comment]
I am getting following error while running the code.

import path from 'path';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at wrapSafe (internal/modules/cjs/loader.js:1047:16)
    at Module._compile (internal/modules/cjs/loader.js:1097:27)
    at loader (C:\SourceCode\msteams-app\node_modules\babel-register\lib\node.js:144:5)
    at Object.require.extensions.<computed> [as .js] (C:\SourceCode\msteams-app\node_modules\babel-register\lib\node.js:154:7)
    at Module.load (internal/modules/cjs/loader.js:977:32)
    at Function.Module._load (internal/modules/cjs/loader.js:877:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at Object.<anonymous> (C:\SourceCode\msteams-app\node_modules\babel-cli\lib\_babel-node.js:154:22)
    at Module._compile (internal/modules/cjs/loader.js:1133:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)

Can anyone please help to resolve.

**Comment by pierregober** [comment]
I was having this same issue. I ended up cloning a forked repo. https://github.com/jrkerns/msteams-samples-hello-world-nodejs

This one works after you install the necessary dependencies (npm install).

You will run into an issue with gulp 3 syntax issues when trying to create the app package but I got this resolved asking around on StackOverflow. Check out this:
https://stackoverflow.com/questions/61642647/i-get-assertionerror-err-assertion-task-function-must-be-specified-when-i-r
